docs/src/main/sphinx/connector/hive.rst

```diff
@@ -984,6 +984,85 @@ Procedures
   Flush Hive metadata cache entries connected with selected partition.
   Procedure requires named parameters to be passed
 
+Table properties
+----------------
+
+.. list-table:: Hive connector storage table properties
+  :widths: 20, 60, 20
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default
+  * - ``auto_purge``
+    - The skip trash when table or partition is deleted.
```

Comment:
This is only prespcriptive. It's up to the Hive Metastore to honor or not honor this. For example Glue doesn't have this concept at all and different vendors/versions of HMS might have different behaviours.

Reply:
Reword as `Indicate to the Hive metastore being used to skip trash when table or partition is deleted.` (i.e. this table property isn't acted upon by Trino at all).